patches/chrome-browser-ui-webui-theme_source.cc.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
+index f3681500ad54c2dcc0a01ddf606c7088a8c5c9d4..6c08f9c33fc744ca61599d150370ebd36e37f016 100644
+--- a/chrome/browser/ui/webui/theme_source.cc
++++ b/chrome/browser/ui/webui/theme_source.cc
+@@ -30,6 +30,10 @@
+ #include "ui/gfx/image/image_skia_rep.h"
+ #include "url/gurl.h"
+ 
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++#include "brave/grit/brave_theme_resources.h"
```

Comment:
This can be removed by overriding similar to `brave_generated_resources.h`.
Will check after finishing rebased build.

Reply:
Done.